client/src/styles/Home.styles.ts

```diff
@@ -56,22 +55,42 @@ export const imageStyles = {
 
 export const innerContainer = {
   root: {
-    width: '37.5rem',
+    maxWidth: '37.5rem',
+    width: '100%',
     marginTop: '3.8125rem'
   }
 };
 
-export const buttonStyle = {
+export const bookAppointmentButtonStyle = {
+  root: {
+    borderColor: '#E1DFDD',
+    fontSize: '1rem',
+    marginRight: '.5rem',
+    marginBottom: '.5rem'
+  }
+};
+
+export const joinLinkButtonStyles = {
   root: {
-    width: '11.8125rem',
     borderColor: '#E1DFDD',
-    fontSize: '.75rem'
+    fontSize: '1rem'
   }
 };
 
+export function calendarIconStyles(theme: PartialTheme | Theme | undefined): IIconProps {
+  return {
+    iconName: 'Calendar',
+    styles: {
+      root: { color: theme?.palette?.themePrimary }
```

Comment:
I think once we verified that `Theme` can not be `undefined`, then we can remove `?`here.

Reply:
done!